src/Hypsography/Hypsography.jl

```diff
@@ -0,0 +1,11 @@
+module Hypsography
```

Comment:
How long did it take you to find this? 😄

Reply:
It's the GiS name for it so I may have come crossed it before lol